frontend/lib/onboarding/routes.tsx

```diff
@@ -37,18 +37,10 @@ export default class OnboardingRoutes extends React.Component<
             signupIntent={signupIntent}
           />
         ),
-        isComplete: (s) =>
-          !!(
-            s.onboardingScaffolding &&
-            s.onboardingScaffolding.firstName &&
-            s.onboardingScaffolding.borough
-          ),
       },
       {
         path: routes.step3,
         render: () => <OnboardingStep3 routes={routes} />,
-        isComplete: (s) =>
-          !!(s.onboardingScaffolding && s.onboardingScaffolding.leaseType),
```

Comment:
This wasn't required, but I wanted to make sure that no steps were ever skipped during onboarding.  I think I originally implemented this back in 2018 in an over-eager attempt to be as user-friendly as possible, but in practicality I don't think it's ever really encountered--skipping steps would only be triggered if e.g. the user completed some steps of onboarding, then got distracted and went somewhere else, and somehow wound up at the LOC splash page again and clicked the CTA button.  At which point we probably _want_ to have them review the information they entered previously, since at this point they might have forgotten what they previously entered.

Reply:
Yep, I think this is both helpful and respectful